monai/apps/auto3dseg/auto_runner.py

```diff
@@ -181,58 +180,80 @@ def __init__(
         self,
         work_dir: str = "./work_dir",
         input: Union[Dict[str, Any], str, None] = None,
-        analyze: bool = True,
-        algo_gen: bool = True,
-        train: bool = True,
-        hpo: bool = False,
-        hpo_backend: str = "nni",
+        algos: Optional[Union[Tuple, List]] = None,
```

Comment:
if we keep the original order of the arguments it'll cause less backward compatibility issues

Reply:
okay, I've rearranged these parameters back as before.  But isn't generally the rule to have input parameters in order of importance and frequency of use?  that's why I though we can move the hpo related parameters toward the end

Reply:
strange, this doesn't seems to be address in this PR, could you please confirm @myron ?

Reply:
I moved `algos` to the end in #5557 and please see my comment in  https://github.com/Project-MONAI/MONAI/pull/5555#issuecomment-1323471371

Reply:
I did change the order of the parameters back to the original, except for new parameter "algo". I though that's what you're asking.   Previously I added "algo" and re-arranged the arguments, but later re-arranged them back, and only added "algo" (new option). 

The order of the parameters should be generally in order of importance, with the most commonly used parameters at the very beginning, and we strive for cleaner more intuitive API.
Auto3dseg is very new code, so it's not too late to make it correct (in terms of parameters).  Any callers to Autorunner is a main class with many options, all callers  should user key-value calls, and not positional arguments , which is a dangerous practice for such large classes.